OpenRA.Game/Graphics/Sheet.cs

```diff
@@ -49,7 +49,7 @@ public Sheet(SheetType type, ITexture texture)
 
 		public Sheet(SheetType type, string filename)
 		{
-			using (var stream = GlobalFileSystem.Open(filename))
+			using (var stream = File.Exists(filename) ? File.OpenRead(filename) : Game.ModData.ModFiles.Open(filename))
```

Comment:
This looks a bit iffy.  What case causes `Game.ModData.ModFiles.Open` to fail?

It might be cleaner to pass a `Stream` into the ctor instead of a filename, and then this logic can be special-cased in the couple of cases that need it.